keps/sig-node/2413-seccomp-by-default/README.md

```diff
@@ -0,0 +1,381 @@
+    `seccomp_use_default_when_empty`
+
+This can result in a behavioral change when doing cluster upgrades while runtime
+administrates may have to take action if they enable the feature.
+
+## Design Details
+
+The feature gate `SeccompDefault` will ensure that the API graduation can be
+done in the standard Kubernetes way. The implementation will be mainly a switch
+from `Unconfined` to `RuntimeDefault`.
+
+Documentation around the feature will be added to the [k/website seccomp
+section](https://kubernetes.io/docs/tutorials/clusters/seccomp).
+
+### Test Plan
+
+There will be unit tests for the feature, whereas the existing seccomp tests can
+be extended to cover the new behavior if enabled.
+
+### Graduation Criteria
+
+#### Alpha
+
+- [ ] Implement the new feature gate
+- [ ] Ensure proper tests are in place
+- [ ] Update documentation to make the feature visible
+
+#### Alpha to Beta Graduation
+
+- [ ] Enable the feature per default
+- [ ] No major bugs reported in the previous cycle
+
+#### Beta to GA Graduation
+
+- [ ] Allowing time for feedback (3 releases)
+- [ ] Risks have been addressed by every common container runtime
+
+### Upgrade / Downgrade Strategy
+
+It's recommended to test the existing workloads with the `RuntimeDefault`
+profile before turning the feature on. Beside that, the feature can be enabled
+on a per node basis to reduce the risk of failing production workloads.
+
+### Version Skew Strategy
+
+There is no explicit version skew strategy required because the feature acts as
+a toggle switch.
+
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+_This section must be completed when targeting alpha to a release._
+
+- **How can this feature be enabled / disabled in a live cluster?**
+
+  - [x] Feature gate (also fill in values in `kep.yaml`)
+    - Feature gate name: `SeccompDefault`
+    - Components depending on the feature gate: `none`
+
+- **Does enabling the feature change any default behavior?**
+
+  Yes, it will change the `Unconfined` seccomp profile to `RuntimeDefault` if
+  no profile is specified.
+
+- **Can the feature be disabled once it has been enabled (i.e. can we roll back
+  the enablement)?**
+
+  Yes, the feature can be disabled but workloads have to be restarted to apply
+  the previous behavior.
+
+- **What happens if we reenable the feature if it was previously rolled back?**
+
+  It will enable the feature again but only apply the new profile to new/restarted
+  workloads.
+
+- **Are there any tests for feature enablement/disablement?**
+
+  Yes, the behavior can be tested via unit tests.
+
+### Rollout, Upgrade and Rollback Planning
+
+_This section must be completed when targeting beta graduation to a release._
+
+- **How can a rollout fail? Can it impact already running workloads?**
+  Try to be as paranoid as possible - e.g., what if some components will restart
+  mid-rollout?
+
+- **What specific metrics should inform a rollback?**
```

Comment:
While not required for alpha, I strongly encourage you to consider this.  If we enable this, how will we assess how well it is working?  Even in alpha, knowing whether it is going well is helpful for admins assessing it and for developers trying to decide how to evolve it.